packages/cache/src/stylis-plugins.js

```diff
@@ -167,9 +167,11 @@ export let createUnsafeSelectorsAlarm = cache => (element, index, children) => {
   )
 
   if (unsafePseudoClasses && cache.compat !== true) {
-    const prevElement = index > 0 ? children[index - 1] : null
-    if (prevElement && isIgnoringComment(last(prevElement.children))) {
-      return
+    for (let i = 0; i < children.length; i++) {
+      const element = children[i]
+      if (element && isIgnoringComment(last(element.children))) {
+        return
+      }
     }
```

Comment:
Could you describe the change? What was the assumption of the previous code? How the fixed case is different and why it didn't work with the previous logic? What was the thought process behind the introduced change? I would appreciate if you could show the AST layouts of different cases so it would be easier to reason about this fix.

Reply:
In the previous logic, only the first ignoreFlag in a style worked.
Therefore, CI passed, but if there were multiple styles and the second or later style used unsafe pseudo classes (ex. `:first-child`), an error would be displayed even if the ignoreFlag was given.